torch/testing/_internal/common_methods_invocations.py

```diff
@@ -510,6 +515,12 @@ def method_tests():
         ('sinh', (), NO_ARGS, 'scalar', (True,)),
         ('cosh', (S, S, S), NO_ARGS, '', (True,)),
         ('cosh', (), NO_ARGS, 'scalar', (True,)),
+        ('conj', (S, S, S), NO_ARGS),
```

Comment:
Do you want to test conj with complex inputs as well?

Reply:
it's being tested with both real and complex inputs since it's added in the list `complex_list`